controllers/che/backup.go

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package che
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	semver "github.com/blang/semver/v4"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	chev1 "github.com/eclipse-che/che-operator/api/v1"
+	"github.com/eclipse-che/che-operator/pkg/deploy"
+)
+
+const (
+	DefaultBackupServerConfigLabelKey = "che.eclipse.org/default-backup-server-configuration"
```

Comment:
`che.eclipse.org/manage-backup-before-update: "true"` ?

Reply:
Do we really need to check the value? Should we accept `True`, `1` or similar as well?

Reply:
sure

Reply:
what about this `che.eclipse.org/backup-before-update: "true"`

Reply:
ok